docs/manual/external-tools.tex

```diff
@@ -537,6 +538,34 @@ \section{Javac compiler\label{javac-installation}}
 % \end{itemize}
 
 
+\section{Lombok\label{lombok}}
+
+Project Lombok (\url{https://projectlombok.org/}) is ``a java library that
+automatically plugs into your editor and build tools, spicing up your java.''
+Starting with version 1.18.3 it is generally easy to combine Lombok and the
```

Comment:
What does it mean to "combine them".  Do you mean to use both, or some deeper integration?  Or do you mean to type-check the generated Lombok code, or to type-check clients of Lombok code?

Reply:
To type-check clients of Lombok code. I'll clarify.